spyder/preferences/configdialog.py

```diff
@@ -1143,8 +1084,8 @@ def _save_lang(self):
 
 
 class ColorSchemeConfigPage(GeneralConfigPage):
-    CONF_SECTION = "color_schemes"
-    NAME = _("Syntax coloring")
+    CONF_SECTION = "appearance"
+    NAME = _("Appearance")

```

Comment:
The problem with calling it "Appearance" is that the first tab of General is *also* called Appearance. Therefore, we either need to revert the name to Themes or the like, or just change the name of that tab to e.g. "Interface". The latter would probably make more sense and I do prefer the "Appearance" name that @ccordoba12 suggested over "Themes" since its more clear.